ClientGUI/INItializableWindow.cs

```diff
@@ -205,6 +206,11 @@ protected void ReadINIForControl(XNAControl control)
                     if (kvp.Value == "Disable")
                         control.LeftClick += (s, e) => Disable();
                 }
+                else if (kvp.Key == "$Text")
+                {
+                    control.Text = section.GetStringValue(nameof(control.Text), string.Empty)
+                        .L10N($"UI:ExtraControl:{kvp.Value}");
+                }
```

Comment:
Would this produce the translation calls for *every* INItializableWindow child that has text overridden or just for extra controls? because it looks like the prefix `ExtraControl` is misleading here.